src/color.rs

```diff
@@ -243,6 +251,8 @@ impl Colors {
         // INode
         m.insert(Elem::INode { valid: true }, Colour::Fixed(13)); // Pink
         m.insert(Elem::INode { valid: false }, Colour::Fixed(245)); // Grey
+        m.insert(Elem::Links { valid: true }, Colour::Fixed(134));
+        m.insert(Elem::Links { valid: false }, Colour::Fixed(222));
```

Comment:
We could reuse the `Inode` colors
```suggestion
        m.insert(Elem::Links { valid: true }, Colour::Fixed(13));
        m.insert(Elem::Links { valid: false }, Colour::Fixed(245));
```